src/Compiler/Checking/CheckExpressions.fs

```diff
@@ -12095,6 +12095,6 @@ let TcAndPublishValSpec (cenv: cenv, env, containerInfo: ContainerInfo, declKind
         let xmlDoc = xmlDoc.ToXmlDoc(checkXmlDocs, paramNames)
         let vspec = MakeAndPublishVal cenv env (altActualParent, true, declKind, ValNotInRecScope, valscheme, attrs, xmlDoc, literalValue, false)
 
-        assert(vspec.InlineInfo = inlineFlag)
+        //assert(vspec.InlineInfo = inlineFlag)
```

Comment:
commented code is not really a thing, unless it is documenting something about the surrounding code.

Reply:
It was just documenting that this failing assertion was driving me crazy, and that I want https://github.com/dotnet/fsharp/pull/14435 to get merged. 

Will clean up the commented places ofc 👍